lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -607,6 +608,28 @@ class DocumentImpl extends NodeImpl {
     });
   }
 
+  // https://dom.spec.whatwg.org/#dom-document-createcdatasection
+  createCDATASection(data) {
+    if (data.indexOf("[[>") !== -1) {
```

Comment:
@brettz9 saw it again just now... 
![capture d ecran 2016-11-08 a 00 38 32](https://cloud.githubusercontent.com/assets/38223/20088092/e0840666-a54b-11e6-943d-cb52450f2258.png)